eng/pipelines/templates/steps/build.yml

```diff
@@ -84,8 +84,8 @@ steps:
         $artifactName = $artifact.name
         Write-Host "Copying $artifactName artifacts to $(Build.ArtifactStagingDirectory)/$artifactName"
         New-Item -Type Directory -Name $artifactName -Path $(Build.ArtifactStagingDirectory) > $null
-        Copy-Item sdk/${{parameters.ServiceDirectory}}/**/$artifactName-*.tgz $(Build.ArtifactStagingDirectory)/$artifactName
-        Copy-Item sdk/${{parameters.ServiceDirectory}}/**/browser/$artifactName-*.zip $(Build.ArtifactStagingDirectory)/$artifactName
+        Copy-Item sdk/${{parameters.ServiceDirectory}}/**/$artifactName-[0-9]*.[0-9]*.[0-9]*.tgz $(Build.ArtifactStagingDirectory)/$artifactName
+        Copy-Item sdk/${{parameters.ServiceDirectory}}/**/browser/$artifactName-[0-9]*.[0-9]*.[0-9]*.zip $(Build.ArtifactStagingDirectory)/$artifactName
```

Comment:
This will technically match something like `artifact-0.1.` or `artifact-0..` or `artifact-ff`

Reply:
only for those starts with number immediately after artifact since this is path filter. right? artifact-ff won't be eligible.

![image](https://user-images.githubusercontent.com/55455725/124314710-cc7bfe00-db40-11eb-844c-456a3898e478.png)